src/pages/Login.js

```diff
@@ -67,59 +72,80 @@ class Login extends Component {
     const { getFieldDecorator } = this.props.form;
     return (
       <UserContext.Consumer>
-      {({isLoading,isLoggedIn}) => {
-        if(isLoggedIn) {
-          this.props.history.push("/");
-        }else{
-          return(
-            <AppLayout>
-              <Card className="login-form-card">
-                <Form onSubmit={this.handleSubmit} className="login-form">
-                  <Form.Item>
-                    {getFieldDecorator('userName', {
-                      rules: [{ required: true, message: 'Please input your username!' }],
-                    })(
-                      <Input prefix={ <Icon type="user"
-                                            style={{ color: 'rgba(0,0,0,.25)' }} />}
-                            placeholder="username"
-                            onChange = {this.usernameOnchange}
-                      />
-                    )}
-                  </Form.Item>
-                  <Form.Item>
-                    {getFieldDecorator('password', {
-                      rules: [{ required: true, message: 'Please input your Password!' }],
-                    })(
-                      <Input prefix={ <Icon type="lock"
-                                            style={{ color: 'rgba(0,0,0,.25)' }} />}
-                        type="password"
-                        placeholder="password"
-                        onChange = {this.passwordOnchange}
-                      />
-                    )}
-                  </Form.Item>
-                  {this.state.error?<div className="red-6">{this.state.error}</div>:null}
-                  <Form.Item>
-                    {getFieldDecorator('remember', {
+        {({ isLoading, isLoggedIn }) => {
+          if (isLoggedIn) {
+            this.props.history.push('/');
+          } else {
+            return (
+              <AppLayout>
+                <Card className="login-form-card">
+                  <Form onSubmit={this.handleSubmit} className="login-form">
+                    <Form.Item>
+                      { getFieldDecorator('userName', {
+                        rules: [{ required: true, message: 'Please input your username!' }],
+                      })(
+                        <Input
+                          prefix={(
+                            <Icon
+                              type="user"
+                              style={{ color: 'rgba(0,0,0,.25)' }}
+                            />
+)}
+                          size="large"
+                          placeholder="Username"
```

Comment:
Change the term "username" to "email". We currently don't have the concept of username.
Also if possible, update the validate stage to check if the user enter a valid email address.